docs/markdown/Python-module.md

```diff
@@ -238,6 +238,63 @@ module.
 
 It exposes the same methods as its parent class.
 
+## A note about Debian installation paths
+
+Debian and derivatives (Ubuntu, Linux Mint etc.) [change default installation
+and import paths][debian-dist-packages] in system interpreter. The problem is,
+the way they do this is inconsistent with documented ways of discovering those
+paths inside Python. Specifically, the [`sysconfig` module][python-sysconfig]
+returns values that are not in [`sys.path`][python-sys-path]. This module is
+used internally by Meson and therefore the installation paths . But it is
+generally expected that if you configure build with `--prefix=/usr` or
+`--prefix=/usr/local` (the latter is the default), Python modules should be
+available without adjusting [`$PYTHONPATH`][python-pythonpath]. Therefore,
+a&nbsp;workaround is needed. There are two, non mutually exclusive options:
+
+### option to override
+
+An option in `meson_options.txt` might allow overriding the value obtained from
+[][`get_install_dir()`], and the files are installed using `install_data()`.
```

Comment:
Is this a missing a reference-link name?

Reply:
I don't understand. Do you mean get_install_dir()? It works, I copied the approach from elsewhere in this module.

Reply:
https://hotdoc.github.io/syntax-extensions.html#referencing-a-symbol

> The label can be empty, in which case it will be set to the name of the symbol at format time